docs/configuration/index-config.md

```diff
@@ -321,7 +328,7 @@ expand_dots: false
 | `description` | Optional description for the field. | `None` |
 | `stored`    | Whether value is stored in the document store | `true` |
 | `indexed`   | Whether value is indexed | `true` |
-| `fast`   | Whether value is fast | `false` |
+| `fast`     | Whether value is stored in a fast field. The default behaviour for text is to store the original text unchanged. It can be configured via `{"normalizer": "lowercase"}`. ([See normalizers](#description-of-available-normalizers)) for a list of available normalizers. | `true` |
```

Comment:
I think we need an entire example here. 
(That's where we pay for the idea of accepting bool and object in the same place)

```yaml
fast:
    normalize: lowercase
```

Reply:
This would be an alternative API.
I think it's nice when having both `enabled` and `normalizer`, but `enabled` on it's own is a little strange.
What do you think?

```yaml
fast:
    enabled: true
    normalizer: lowercase
```
```yaml
fast:
    enabled: true
```
```yaml
fast:
    enabled: false
```

Reply:
Yes, I'm not fond of having `enabled: false` alone. But we then need two examples.

Reply:
We also have other field types that can't have a normalizer, which are configured with `fast: true`